packages/expo-font/ios/FontUtils.swift

```diff
@@ -58,7 +58,7 @@ internal func registerFont(_ fontUrl: CFURL) throws {
       // - another instance already registered with the same name (assuming it's most likely the same font anyway)
       return
     default:
-      throw FontRegistrationFailedException(error)
+      throw FontRegistrationFailedException(FontRegistrationErrorInfo(fontFamilyAlias: fontFamilyAlias, cfError: error, ctFontManagerError: fontError))
```

Comment:
How would the error message look like if you use chained errors? Something like
```suggestion
      throw FontRegistrationFailedException().causedBy(error)
```
This should then contain the entire description of the native error in the final error message.

Reply:
@tsapeta the result is 

```
Calling the 'loadAsync' function has failed
→ Caused by: Registering 'sfdds' font failed.
→ Caused by: Font registration was unsuccessful.
```

the native error's description doesn't contain the error code, it has to be extracted using `CFErrorGetCode()`